aks/postgres/variables.tf

```diff
@@ -44,6 +44,12 @@ variable "cluster_configuration_map" {
   description = "Configuration map for the cluster"
 }
 
+variable "server_docker_image" {
+  type        = string
+  default     = "postgres:14-alpine"
+  description = "Database image to use with kubernetes deployment"
```

Comment:
Can you add the postgis example in the description? This is a very commoon use case here